app/src/main/java/com/safetyheads/akademiaandroida/MainActivity.kt

```diff
@@ -125,4 +94,22 @@ class MainActivity : AppCompatActivity() {
             commit()
         }
     }
+
+    private fun fetchDataFromRemoteConfig() {
```

Comment:
Name of this function suggest that something will be fetched but you only here observe on config change or failure

Reply:
Yes, it came into my mind after i did commit... my bad....
I saw Albert PR, and I think he has named it very well, something like "initObserver" would be nice